tests/unit/python/foglamp/common/storage_client/test_exceptions.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test foglamp/common/storage_client/exceptions.py """
+
+import pytest
+
+from foglamp.common.storage_client.exceptions import StorageClientException
+from foglamp.common.storage_client.exceptions import *
+
+__copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("common", "storage_client")
+class TestStorageClientExceptions:
+
+    def test_init_StorageClientException(self):
+
+        with pytest.raises(Exception) as excinfo:
+            raise StorageClientException()
```

Comment:
If no arg passed, remove parentheses, works well without it

Reply:
this is an explicit class for internal use to subclass. Hence an explicit class test